src/Sylius/Component/Core/Test/Services/DefaultCountriesFactoryInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Test\Services;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface DefaultCountriesFactoryInterface
```

Comment:
It looks more like a provider, it also adds them to repository. The "default" keyword is not needed here IMO.

Also, why don't we create `CountryProviderInterface::provide($countryCode)` that:
  - returns Country object if it exists in the repository
  - creates a new Country and adds it into repository

This way we get rid of that array parameter that doesn't look so good and we won't miss the additional features that comes with this "factory". It will also allow us to do something more with that Country object.